back/app/policies/user_policy.rb

```diff
@@ -76,6 +76,10 @@ def comments_count?
     true
   end
 
+  def participation_stats?
+    record.id == user&.id || (active? && !user.normal_user?)
```

Comment:
Should project moderators be able to view participation stats of everyone?

Reply:
Good catch! I thought I changed that :)